gitops/prepare/prepare.go

```diff
@@ -29,11 +29,6 @@ import (
 const (
 	// namespace where the bundle configuration will be searched in case it is not found in the component's namespace
 	BuildBundleDefaultNamespace = "build-templates"
```

Comment:
can be removed as well:
BuildBundleDefaultNamespace, AppStudioFallbackBuildBundle, HACBSFallbackBuildBundle

Reply:
Done